i18n/en.pot

```diff
@@ -428,6 +428,9 @@ msgstr ""
 msgid "Indicators Synchronization"
 msgstr ""
 
+msgid "Select children subtree"
+msgstr ""
```

Comment:
Isn't a bit misleading, as it also selects the parent org unit?

Reply:
Maybe "Select with children subtree"